UPDATING.md

```diff
@@ -42,6 +42,7 @@ assists people when migrating to a new version.
 
 ### Other
 
+- [17589](https://github.com/apache/incubator-superset/pull/17589): It is now possible to limit access to usersä recent activity data by setting the `ENABLE_PUBLIC_ACTIVITY_ACCESS` config flag to false, or customizing the `raise_for_user_activity_access` method in the security manager.
```

Comment:
Typo - usersä 🙂

Reply:
Lol, that's what you get jumping between FIN and US keyboard layouts 1000x per day 😆